CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ option (DO_INCLUDE_PLUGINS "Build subproject plugins" OFF)
 option (DO_INCLUDE_SDK "Build subproject sdk-cpp" OFF)
 
 option (DO_BUILD_TESTS "Set DO_BUILD_TESTS to OFF to skip building tests." ON)
+option (DO_ENABLE_EXCEPTIONS "Set DO_ENABLE_EXCEPTIONS to ON to disable compiling with exceptions" OFF)
```

Comment:
>DO_ENABLE_EXCEPTIONS [](http://example.com/codeflow?start=8&length=20)

Since this applies to the sdk only, how about moving all changes in this file to the sdk cmakelists.txt?

Reply:
I see this is copied to sdk-cpp/cmakelists.txt. Delete from here?